scalafmt-core/shared/src/main/scala/org/scalafmt/internal/FormatTokens.scala

```diff
@@ -135,6 +135,8 @@ class FormatTokens(leftTok2tok: Map[TokenOps.TokenHash, Int])(
   def getLastNonTrivialOpt(tree: Tree): Option[FormatToken] =
     TokenOps.findLastNonTrivialTokenOpt(tree.tokens).map(apply)
 
+  @inline
+  def tokenAfter(token: Token): FormatToken = nextNonComment(before(token))
```

Comment:
Does it really look for token after? `before(token)` is the one before no? It's a bit unclear what this function should actually do.

Reply:
they both find the token *BUT* if it had been rewritten and is no longer there, `before` returns the token just before it and `after` just after it.

`tokenAfter` returns the first format token such that `right` is to the right of `token` and is not a comment. there are several similar methods below, this one was missing before (while tokenBefore(token) was used already).

Reply:
added a commit which documents these methods. ptal.